Client/Application/LeanplumIntegration.swift

```diff
@@ -248,13 +217,13 @@ class LeanPlumClient {
         self.setupCustomTemplates()
         lpState = .willStart
         // To be removed after after beta testing - userID value
-        Leanplum.start(withUserId: userID, userAttributes: attributes, responseHandler: { _ in
+        Leanplum.start(withUserId: nil, userAttributes: attributes, responseHandler: { _ in
```

Comment:
nit: `assert(Thread.isMainThread)` would help the reader here, because that is what I assume, but maybe LeanPlum isn't following common convention of callbacks being on main.

Reply:
On LP's documentation they don't mention anything about being run on main thread. Not sure if adding this will either make or break LP integration